.changeset/wet-chefs-develop.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/vercel': patch
+'@astrojs/node': patch
+---
```

Comment:
Mmmmh, shouldn't this be a `minor`? We are adding a new feature :)

Reply:
IIRC patches are fine for experimental features in core but idk what should be done for other official packages relying on experimental features! Is that documented anywhere?

Reply:
I think It should be a minor because they both require a new feature exposed by Astro that was available in 4.10 (Astro features). Also, we released the experimental feature in a minor, not a patch, hence adapters should do the same. 

However, **changes in _existing_** experimental features are accepted in patches.

Hence, I'd say these changes should be shipped as a minor because we ship a new user facing feature.